plugins/modules/podman_secret.py

```diff
@@ -78,43 +78,33 @@
 from ansible.module_utils.basic import AnsibleModule
 
 
-def is_not_exists(err):
-    return 'no such secret' in err
-
-
 def podman_secret_create(module, executable, name, data, force):
     if force:
-        rc, out, err = module.run_command([executable, 'secret', 'rm', name])
-        if rc == 0:
-            pass
-        elif not is_not_exists(err):
-            module.fail_json(msg="Unable to remove secret: %s" % err)
+        module.run_command([executable, 'secret', 'rm', name])
 
-    rc, out, err = module.run_command(
+    rc, _, err = module.run_command(
         [executable, 'secret', 'create', name, '-'], data=data)
 
     if rc != 0:
         module.fail_json(msg="Unable to create secret: %s" % err)
 
     return {
         "changed": True,
-        "id": out,
     }
 
 
 def podman_secret_remove(module, executable, name):
     changed = False
-    rc, out, err = module.run_command([executable, 'secret', 'rm', name])
+    rc, _, err = module.run_command([executable, 'secret', 'rm', name])
```

Comment:
pylint in ansible-test complains about "_": blacklisted-name: Black listed name "_"